<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>amphetype2</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="Quizzer" name="quizzer">
       <attribute name="title">
        <string>Typer</string>
       </attribute>
      </widget>
      <widget class="PerformanceHistory" name="performanceHistory">
       <attribute name="title">
        <string>Performance</string>
       </attribute>
      </widget>
      <widget class="StatisticsWidget" name="statisticsWidget">
       <attribute name="title">
        <string>Analysis</string>
       </attribute>
      </widget>
      <widget class="LessonGenWidget" name="lessonGenWidget">
       <attribute name="title">
        <string>Lesson Generator</string>
       </attribute>
      </widget>
      <widget class="TrainingGenWidget" name="trainingGenWidget">
       <attribute name="title">
        <string>Training Generator</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_Plot"/>
    <addaction name="action_Library"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="plotDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>38</width>
     <height>140</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string>plot</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="LivePlot" name="plot"/>
  </widget>
  <action name="action_Plot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Plot</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_Library">
   <property name="text">
    <string>&amp;Library</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Quizzer</class>
   <extends>QWidget</extends>
   <header>quizzer/quizzer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PerformanceHistory</class>
   <extends>QWidget</extends>
   <header>performance/performancehistory.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatisticsWidget</class>
   <extends>QWidget</extends>
   <header>analysis/statisticswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrainingGenWidget</class>
   <extends>QWidget</extends>
   <header>generators/traininggenwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LivePlot</class>
   <extends>QWidget</extends>
   <header>liveplot/liveplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LessonGenWidget</class>
   <extends>QWidget</extends>
   <header>generators/lessongenwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
